run_nomad.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+HOST_IP=$(ip route get 8.8.8.8 | awk '{print $NF; exit}')
+
+echo "Rebuilding Docker image while waiting for Nomad to come online."
+
+# Start the nomad in development locally
+nomad agent -bind $HOST_IP \
+      -data-dir /home/kurt/Development/data_refinery/nomad_dir \
+      -dev \
+      > nomad.logs &
+
+docker build -t wkurt/nomad-test:second -f workers/Dockerfile .
+
+check_nomad_status () {
+    echo $(curl --write-out %{http_code} \
+                  --silent \
+                  --output /dev/null \
+                  http://$HOST_IP:4646/v1/status/leader)
+}
+
+# Wait for Nomad to get started.
+nomad_status=$(check_nomad_status)
+while [ $nomad_status != "200" ]; do
+    sleep 0.5
```

Comment:
Do you know how portable this `sleep` command is? I know earlier Unix/Linux implementations only accept an integer. If it is not portable, then `sleep 1` is probably better.